resources/visualizations/Application-Tags-by-Bandwidth.json

```diff
@@ -0,0 +1,9 @@
+{
+   "visState": "{\"type\":\"pie\",\"params\":{\"shareYAxis\":true,\"addTooltip\":true,\"addLegend\":true,\"isDonut\":true},\"aggs\":[{\"id\":\"1\",\"type\":\"sum\",\"schema\":\"metric\",\"params\":{\"field\":\"TotalBytesDelta\"}},{\"id\":\"2\",\"type\":\"terms\",\"schema\":\"segment\",\"params\":{\"field\":\"ApplicationFamily.raw\",\"size\":10,\"order\":\"desc\",\"orderBy\":\"1\"}},{\"id\":\"4\",\"type\":\"terms\",\"schema\":\"segment\",\"params\":{\"field\":\"Application\",\"size\":10,\"order\":\"desc\",\"orderBy\":\"1\"}}],\"listeners\":{}}", 
+   "kibanaSavedObjectMeta": {
+      "searchSourceJSON": "{\"index\":\"[network_]YYYY_MM_DD\",\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}},\"filter\":[]}"
+   }, 
+   "version": 1, 
+   "description": "", 
+   "title": "Application Tags by Bandwidth"
```

Comment:
Sorry to keep finding things individually, I wish I would have caught this before putting up my last comment. 

This should be named `Application Families by Bandwidth`

Reply:
@craig-cogdill, I don't know about the recommendation for capitalizing the 'B'. In "The Elements of Style", the recommendation would be to not capitalize prepositions unless it is the first or last word. Brian would know for sure.

Reply:
@john-gress I agree with you 100%, but all of our other visualizations are named that way :'(